### ISSUE TITLE
Details for finetune SQA dataset

### ISSUE THREAD

**Issue by Unrealluver** [haotian-liu/LLaVA#95]
Greetings! I have checked the great performance on your COCO VQA datasets. Now I want to check the performance on the SQA dataset, could you give me more guidance about the fine-tuning details? Is the LLaVA-13b-SQA still fine-tuning on the LLaVA-13b-cc3m-595k? A running script containing epochs, learning rate, and other training parameters helps us followers a lot to reproduce LLaVA. 

Thanks for your time!

**Comment by haotian-liu** [comment]
Hi @Unrealluver, thank you for the interest in our work.  I have updated our README to include [details of ScienceQA training](https://github.com/haotian-liu/LLaVA#fine-tuning-on-scienceqa) to reproduce our current checkpoint (w/o image start/end tokens).
I have also provided the pretrained projectors, please see the link in the instructions.
Please let me know if there are anything unclear, thanks.

**Comment by haotian-liu** [comment]
Closing the issue due to inactivity, please feel free to re-open if you meet any other issues, thanks.
